atlasexec/atlas.go

```diff
@@ -272,11 +281,11 @@ func (c *Client) MigrateApply(ctx context.Context, params *MigrateApplyParams) (
 		args = append(args, strconv.FormatUint(params.Amount, 10))
 	}
 	args = append(args, params.Vars.AsArgs()...)
-	return jsonDecodeErr[MigrateApply, *MigrateApplyError](c.runCommand(ctx, args))
+	return jsonDecodeErr[MigrateApply](newMigrateApplyError)(c.runCommand(ctx, args))
 }
 
-// SchemaApply runs the 'schema apply' command.
-func (c *Client) SchemaApply(ctx context.Context, params *SchemaApplyParams) (*SchemaApply, error) {
+// MultipleSchemaApply runs the 'schema apply' command for multiple targets.
+func (c *Client) MultipleSchemaApply(ctx context.Context, params *SchemaApplyParams) ([]*SchemaApply, error) {
```

Comment:
I suggest naming it as: MigrateApplySlice, by the return type. This way, auto-completion is also friendlier. 

Not the best name, but correct (because the slice can also be empty).

Reply:
This is the new method, not renaming. :D. I'll update them to `*Slice`